data/transactions/logic/assembler.go

```diff
@@ -2269,7 +2269,7 @@ func disassemble(dis *disassembleState, spec *OpSpec) (string, error) {
 	}
 	if strings.HasPrefix(spec.Name, "bytec_") {
 		b := spec.Name[len(spec.Name)-1] - byte('0')
-		if int(b) < len(dis.intc) {
+		if int(b) < len(dis.bytec) {
```

Comment:
How is this change related to the current issue?

Reply:
It's not, but it's a dumb cut-n-paste bug I found while wandering around.